src/examples/common/utils.ts

```diff
@@ -6,6 +6,10 @@ import {
   VisualizationDefinition
 } from '../types/types'
 
+export function log(...args: any[]) {
+  console.log.apply(console, args)
```

Comment:
FWIW, you can also pass splats/spreads in: `console.log(...args)`

Reply:
yeah this is just my `log` shortcut so i don't have to type `console.log` : )
i'll remove this before we send it up to looker.